src/test/java/co/aurasphere/botmill/fb/test/api/MessengerCodeApiTest.java

```diff
@@ -47,12 +53,18 @@
 	
 	@Before
 	public void setup() {
+		StandardPBEStringEncryptor enc = new StandardPBEStringEncryptor();
+		enc.setPassword("password"); // can be sourced out
+		ConfigurationUtils.loadEncryptedConfigurationFile(enc, "botmill.properties");
+
 		FbBotMillContext.getInstance().setup(ConfigurationUtils.getEncryptedConfiguration().getProperty("fb.page.token"), 
 				ConfigurationUtils.getEncryptedConfiguration().getProperty("fb.validation.token"));
 	}
 
 	@Test
 	public void testMessengerCodeApi() {
+		FbBotMillContext.getInstance().setup(ConfigurationUtils.getEncryptedConfiguration().getProperty("fb.page.token"), 
+				ConfigurationUtils.getEncryptedConfiguration().getProperty("fb.validation.token"));
 		MessengerCode response = MessengerCodeApi.getMessengerCode();
```

Comment:
Can we remove the property for testing from the configuration we are committing and put them back into environment variables so other user can't trigger these tests when building in local? Also, if we do that, we need a way to skip if such variables are not set so that the build doesn't break